Dockerfile

```diff
@@ -10,24 +10,23 @@ FROM php:apache
 ENV JQUERY 3.2.1
 ENV CLIPBOARD 1.7.1
 
-RUN apt-get update && apt-get install -y yaz libyaz4-dev wget unzip \
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y yaz libyaz4-dev wget unzip python3-bs4 python3-requests \
     && pecl install yaz \
     && docker-php-ext-enable yaz
 
-WORKDIR /malibu
-RUN mkdir isbn
+WORKDIR /var/www/html/malibu
+
+# Copy the complete directory structure sans entries in .dockerignore
+COPY . .
```

Comment:
Can we move that line a little bit down again? This will create docker to generate a new layer whenever we change any file and therefore, I think, it is better to do this at a later time.

Reply:
This has been addressed by 76f55d2